### PR TITLE
correct BR document

### PR DESCRIPTION
**Pull request by IANTHEREAL**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

correct BR feature description

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by 3pointer — APPROVED** [review]

File: br/use-br-command-line-tool.md [hunk 1]
```diff
@@ -501,14 +501,12 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
 
 > **Warning:**
 >
-> Although you can back up and restore system tables (such as `mysql.tidb`) using the BR tool, some unexpected situations might occur after the restore, including:
+> Although you can back up the system tables (such as `mysql.tidb`) using the BR tool, but BR does not support the restoration of the following system tables. Even if you use the `-f` setting to restore this table, they will still be ignored:
```
Comment (shichun-0415) [inline br/use-br-command-line-tool.md:504]: Suggested change: Although you can back up system tables (such as `mysql.tidb`) using the BR tool, BR ignores the following system tables even if you use the `-f` setting to perform the restoration:

File: br/use-br-command-line-tool.md [hunk 2]
```diff
@@ -501,14 +501,12 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
...
 >
-> - the statistical information tables (`mysql.stat_*`) cannot be restored.
-> - the system variable tables (`mysql.tidb`，`mysql.global_variables`) cannot be restored.
-> - the user information tables (such as `mysql.user` and `mysql.columns_priv`) cannot be restored.
-> - GC data cannot be restored.
->
-> Restoring system tables might cause more compatibility issues. To avoid unexpected issues, **DO NOT** restore system tables in the production environment.
+> - the statistical information tables (`mysql.stat_*`);
+> - the system variable tables (`mysql.tidb`，`mysql.global_variables`);
+> - the user information tables (such as `mysql.user` and `mysql.columns_priv`);
```
Comment (shichun-0415) [inline br/use-br-command-line-tool.md:508]:
```suggestion
> - Statistical information tables (`mysql.stat_*`)
> - System variable tables (`mysql.tidb`，`mysql.global_variables`)
> - User information tables (such as `mysql.user` and `mysql.columns_priv`)
```

File: br/use-br-command-line-tool.md (hunk 1 above)
Comment (IANTHEREAL) [reply br/use-br-command-line-tool.md:504]: yes

File: br/use-br-command-line-tool.md [hunk 3]
```diff
@@ -501,14 +501,12 @@ br restore full -f 'mysql.usertable' -s $external_storage_url --ratelimit 128
...
-> - the statistical information tables (`mysql.stat_*`) cannot be restored.
-> - the system variable tables (`mysql.tidb`，`mysql.global_variables`) cannot be restored.
-> - the user information tables (such as `mysql.user` and `mysql.columns_priv`) cannot be restored.
-> - GC data cannot be restored.
->
-> Restoring system tables might cause more compatibility issues. To avoid unexpected issues, **DO NOT** restore system tables in the production environment.
+> - Statistical information tables (`mysql.stat_*`)
+> - System variable tables (`mysql.tidb`，`mysql.global_variables`)
+> - User information tables (such as `mysql.user` and `mysql.columns_priv`)
+> - [other system tables](https://github.com/pingcap/tidb/blob/v5.3.0/br/pkg/restore/systable_restore.go#L31)
```
Comment (shichun-0415) [inline br/use-br-command-line-tool.md:509]:
```suggestion
> - [Other system tables](https://github.com/pingcap/tidb/blob/v5.3.0/br/pkg/restore/systable_restore.go#L31)
```

**Pull request merged**